src/components/LoginPage/LoginPage.js

```diff
@@ -1,12 +1,37 @@
-import React from 'react';
+import TextInput from '../inputs/TextInput'
+import './Login.scss'
 
-const LoginPage = () => {
+
+export const NewLogin = () =>{
+    function handleSubmit(event) {
+        event.preventDefault()
+        const formData = new FormData(event.target)
+        const formProps = Object.fromEntries(formData)
+        console.log('You logged in!')
+        console.log("formProp", formProps)
```

Comment:
Are we keeping console logs?

Reply:
I can delete this, it was only for testing!